### PR TITLE
Add infrastructure DNS repo

### PR DESCRIPTION
**Pull request by elicoidal**

### CODE REVIEW

File: tools/repos_with_ids.txt [hunk 1]
```diff
@@ -122,3 +122,4 @@
 221|github.com/OCA/l10n-ethiopia
 222|github.com/OCA/business-requirement
 223|github.com/OCA/l10n-cambodia
+223|github.com/OCA/infrastructure-dns
```
Comment (pedrobaeza) [inline tools/repos_with_ids.txt]: It can't be 223, as this one is for l10n-cambodia. Have you already created the repo in runbot?

Comment (elicoidal) [inline tools/repos_with_ids.txt]: sorry it is 224 (stupid copy paste mistake)

**Comment by pedrobaeza** [comment]
OK, we can fast-track squashing.

**Comment by dreispt** [comment]
What about Transifex key?

**Pull request merged**